docs/modules/ROOT/pages/index.adoc

```diff
@@ -939,23 +967,48 @@ The relayer supports two ways to configure RPC URLs:
 
 1. **Public RPC URLs**: These are the default RPC endpoints provided by the network. They are automatically selected based on the network configuration.
 
-2. **Custom RPC URL**: You can specify a custom RPC URL using the `custom_rpc_urls` field in the relayer configuration. This is useful when you want to:
-   * Use your own RPC node
+2. **Custom RPC URLs**: You can specify custom RPC URLs using the `custom_rpc_urls` field in the relayer configuration. Each URL can be configured with an optional weight for high availability:
```

Comment:
Maybe weight should not be optional or we should document what is default weight.

I am thinking about uses case when user defines multiple rpc urls. Few of them could contain weight and few could miss it. In this case it is not clear enough what priority they would take. Our default value could be bigger than weight they used for some urls.

If we force to define weight order would be always clear. If they use max value for all urls it means they have same priority.